test/models/services.ts

```diff
@@ -425,6 +425,62 @@ describe('services.chooseServiceConfig and services.buildService', function () {
       });
     });
   });
+
+  describe('when the collection match includes variables for variable-based service', function () {
+    const collectionId = 'C123-TEST';
+    const variableId = 'V123-TEST';
+    beforeEach(function () {
+      this.config = [
+        {
+          name: 'variable-based-service',
+          type: { name: 'turbo' },
+          capabilities: {
+            subsetting: { variable: true },
+            output_formats: ['text/csv'],
+          },
+          collections: [
+            {
+              [collectionId]: [ variableId ],
+            },
+          ],
+        },
+      ];
+    });
+
+    describe('requesting service with variable subsetting', function () {
```

Comment:
You don't need to do this in the current PR, but I have some notes for all the tests I'd like to see when we fully integrate the giovanni service.

1. One variable requested and matches (done)
2. Request for 'all' - aka no variable subsetting requested (done)
3. Two variables provided and both match
4. Two variables provided and one of the two match
5. One variable provided and doesn't match

Reply:
Sure, I will add them in some following giovanni stories ... thanks!

Reply:
Done in story 1061.